core/src/main/kotlin/fr/sncf/osrd/api/api_v2/standalone_sim/SimulationRequest.kt

```diff
@@ -143,5 +143,6 @@ class SimulationPowerRestrictionItem(
 )
 
 class TrainScheduleOptions(
-    @Json(name = "use_electrical_profiles") val useElectricalProfiles: Boolean
+    @Json(name = "use_electrical_profiles") val useElectricalProfiles: Boolean,
+    @Json(name = "ignore_infra_speed_limits") val ingoreInfraSpeedLimits: Boolean
```

Comment:
We may want to invert the logic to be consistent with `use_electrical_profiles`. We could make it optional as well, and use them if unspecified.